### PR TITLE
added style to image-search

### PR DESCRIPTION
**Pull request by iam-benyamin**

### CODE REVIEW

**Comment by darioguarascio** [comment]
Ideal case is to create an Astro component out of this form, that we can use and reuse in different areas with `<SearchForm/>` syntax

**Comment by mr-mahmoudyan** [comment]
I will make it component

File: astro/src/components/SearchForm.astro [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+---
+
+<form
+  enctype="multipart/form-data"
+  action="/api/input/form"
+  method="post"
+  class="border-2 border-color: #dfe1e5; shadow-xl rounded-full flex justify-center items-center pr-3 pl-3"
```
Comment (darioguarascio) [inline astro/src/components/SearchForm.astro]: `border-color: #dfe1e5` is a css style, added in a class attribute @iam-benyamin

File: astro/src/styles/components/SearchForm.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+.light-gray-border {
```
Comment (darioguarascio) [inline astro/src/styles/components/SearchForm.css]: @iam-benyamin sorry to be annoying. whenever possible, try to use a color that is provided by tailwind. If it is not provided, generate a new color serie (https://uicolors.app/create) and add it to the tailwind config file, so you can use it in any class. 
IE: you create a new color called `mygrey` and you can use classes like `text-mygrey-100` or in this case `border-mygrey`

**Review by darioguarascio — APPROVED** [review]
Quite good. Later on the hardcoded actions and strings will be replaced. also `mygray` name sounds silly. but overall ok!

**Pull request merged**